packages/ui/src/hooks/useVehiclesLayer.ts

```diff
@@ -117,11 +139,22 @@ const useVehiclesLayer = ({ route, vehicles, vehiclesLayer }: UseRouteVehiclesLa
             className: `busmap-vehicle ${quad}`,
             html: div
           })
-          const marker = new VehicleMarker(
-            L.latLng(vehicle.lat, vehicle.lon),
-            { vehicleId: vehicle.id, directionId: vehicle.directionId },
-            { icon }
-          )
+          const marker = new VehicleMarker(L.latLng(vehicle.lat, vehicle.lon), vehicle, {
+            icon
+          })
+          const popup = L.popup({
```

Comment:
Set the `offset` to position the popup over the top center of the associated marker.

Reply:
Done in [d54e511](https://github.com/morganney/busmap/pull/20/commits/d54e5113a34855eb5e9cb4756d6d183e8d0d0b59).